src/service/project/updateProject.ts

```
@@ -0,0 +1,108 @@
+import { prisma } from "../../config/prisma";
+import { Response } from "express";
+import { AuthenticatedRequest, BasicResponse } from "../../types";
+import { RegisterProjectBody } from "../../types/project"
+import { validateProjectInput } from "../../utils/validation"
+import { ProjectState } from "../../config/prisma";
+
+// 프로젝트 글 수정
+
+export const updateProject = async (
+  req: AuthenticatedRequest<{}, {}, RegisterProjectBody>,
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)